packages/manager/apps/web/client/app/app.js

```diff
@@ -449,9 +446,6 @@ angular
     set(editableThemes, 'default.submitTpl', ['<button style="background:none;border:none" type="submit">', '<i class="fa fa-check green"></i>', '</button>'].join(''));
     set(editableThemes, 'default.cancelTpl', ['<button style="background:none;border:none" ng-click="$form.$cancel()">', '<i class="fa fa-times red"></i>', '</button>'].join(''));
   })
-  .config((OtrsPopupProvider, constants) => {
```

Comment:
If `constants .REDIRECT_URLS.listTicket` is not used anymore you could remove it from constants.